pkgs/development/python-modules/klaus/default.nix

```diff
@@ -52,6 +53,15 @@ buildPythonPackage rec {
   # Needs to set up some git repos
   doCheck = false;
 
+  # TODO: remove when this is upstreamed
+  patches = [
```

Comment:
`patches` is usually located after `src`

Reply:
I moved them to after `prepatch` so it's chronological-ish